### DISCUSSION TITLE
4.10.2

### DISCUSSION THREAD

**Discussion by jrosdahl** [ccache/ccache#1486] [General]
* [Generic source code release (tar.gz)](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2.tar.gz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2.tar.gz.asc))
* [Generic source code release (tar.xz)](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2.tar.xz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2.tar.xz.asc))
* [Darwin (macOS) universal binary release](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-darwin.tar.gz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-darwin.tar.gz.asc))
* [Linux x86_64 binary release](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-linux-x86_64.tar.xz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-linux-x86_64.tar.xz.asc))
* [Windows i686 binary release](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-windows-i686.zip) ([signature](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-windows-i686.zip.asc))
* [Windows x86_64 binary release](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-windows-x86_64.zip) ([signature](https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-windows-x86_64.zip.asc))
* [Release notes](https://ccache.dev/releasenotes.html#_ccache_4_10_2)
* [Manual](https://ccache.dev/manual/4.10.2.html)

<hr /><em>This discussion was created from the release <a href='https://github.com/ccache/ccache/releases/tag/v4.10.2'>4.10.2</a>.</em>

**Comment by gmesm** [comment]
Is there CI that was used to generate these release binaries? It does not seem that the Github Actions CI was used to generate them based on the actions yml found in this repo. It also doesn't seem that build-in-docker was used to generate them either. I would like to add support for ARM64 Linux given that GitHub actions now has free hosted ARM64 Linux runners for open source repos but cannot do that if CI isn't already being used to generate releases.

**Reply by jrosdahl** [reply]
With one exception: no, the release binaries are not built in GitHub CI. The exception is the macOS binary which is built in the [build_macos_universal](https://github.com/ccache/ccache/blob/82e5444979112535195cf208a7e782c327653a1a/.github/workflows/build.yaml#L254-L273) job. If you contribute an arm64 equivalent job then we could look into incorporating the binary into the release.
